rc.sh

```diff
@@ -16,9 +16,18 @@
 # limitations under the License.
```

Comment:
nit: please mark this file as executable (`chmod +x ./rc.sh`)

Reply:
Updated at: https://github.com/hazelcast/hazelcast-go-client/pull/555/commits/d0d01a8a006e811d076359c801396dbf51361ea8